job-dsls/jobs/downstream_pr_jobs.groovy

```diff
@@ -168,17 +168,25 @@ for (repoConfig in REPO_CONFIGS) {
                         mavenArgs(get("upstreamMvnArgs"))
                     }
                 }
-                project / 'builders' << 'hudson.tasks.Maven' {
-                    mavenName("kie-maven-${Constants.MAVEN_VERSION}")
-                    jvmOptions("-Xms1g -Xmx4g -XX:+CMSClassUnloadingEnabled")
-                    targets("-e -fae -nsu -B -T1C clean install -s \$SETTINGS_XML_FILE -Dkie.maven.settings.custom=\$SETTINGS_XML_FILE -Dfull -DskipTests")
+                if (repo != "kie-wb-common") {
+                    project / 'builders' << 'hudson.tasks.Maven' {
+                        mavenName("kie-maven-${Constants.MAVEN_VERSION}")
+                        jvmOptions("-Xms1g -Xmx4g -XX:+CMSClassUnloadingEnabled")
+                        targets("-e -fae -nsu -B -T1C clean install -s \$SETTINGS_XML_FILE -Dkie.maven.settings.custom=\$SETTINGS_XML_FILE -Dfull -DskipTests")
+                    }
+                } else {
+                    project / 'builders' << 'hudson.tasks.Maven' {
+                        mavenName("kie-maven-${Constants.MAVEN_VERSION}")
+                        jvmOptions("-Xms1g -Xmx7g -XX:+CMSClassUnloadingEnabled")
```

Comment:
I see in https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-stunner/kie-wb-common-stunner-showcase/kie-wb-common-stunner-showcase-project/pom.xml#L1297 there is another try out to have even bigger heap size value limit. So is this expected to be there @romartin ?

Reply:
yeah @mareknovotny @mbiarnes - it's possible to 'override' the settings given in the maven `gwt` plugin's configuration, by using jvm system properties. You can check all those in the [compile goal's reference](https://gwt-maven-plugin.github.io/gwt-maven-plugin/compile-mojo.html).

So for example, the `extraJvmArgs` can be changed from command line by using `-Dgwt.extraJvmArgs='...'`